### DISCUSSION TITLE
All sites deactivated, but still getting Backup messages (and backups :-) )

### DISCUSSION THREAD

**Discussion by drdingens** [akeeba/panopticon#836] [Q&A (Community Support)]
Hey
I have deactivated all sites in Panopticon. 
![2024_10_26_16_43_00_Google Chrome](https://github.com/user-attachments/assets/463736a1-b7be-4461-9d9a-02cc9ea65980)
All the sites are still making their Backup at the time I set in Panopticon and I am still notified about this:

![2024_10_26_16_44_28_Mail](https://github.com/user-attachments/assets/716c5ac4-63a2-45fd-9d2f-50f3118c911a)
Is this intended?
Thanks a lot!
Rolf

**Comment by nikosdion** [comment]
Disabling a site will only hide it from view, and remove it from information collection (core updates, extensions). It will not disable any tasks you have created for the site, including backups.

**Comment by drdingens** [comment]
So I need to delete the sites, or is there another way?
Thanks
Rolf
--------------
http://www.bodhran-info.de
 
Dr. Rolf Wagels
An der Kapelle 10 
31311 Uetze 
mobile: +49-171-3245860

**Reply by nikosdion** [reply]
Administration, Tasks. Type -> Take a backup with Akeeba Backup. Now you see all of the scheduled backup tasks. You can disable those you don't need.

**Comment by drdingens** [comment]
Hey
Thanks!
Rolf
--------------
http://www.bodhran-info.de
 
Dr. Rolf Wagels
An der Kapelle 10 
31311 Uetze 
mobile: +49-171-3245860
